descriptor/qsarmolecular/src/test/java/org/openscience/cdk/qsar/descriptors/molecular/AtomTyperTests.java

```diff
@@ -0,0 +1,458 @@
+package org.openscience.cdk.qsar.descriptors.molecular;
```

Comment:
Please add the copyright/license header here too.

Reply:
Added also removed the unused import of the CDKhydrogenadder